layout.js

```diff
@@ -4,22 +4,47 @@
 (function() {
 
 var logPane = document.getElementById('log'),
-    consolePane = document.getElementById('console-wrapper');
+    consolePane = document.getElementById('console-wrapper'),
+    container = document.getElementById('container');
 
 // Set the pane widths based on drag position.
 document.getElementById('drag-resize-bar').addEventListener('drag', function(event) {
-  if (event.x) {
-    var maxWidth = window.innerWidth - this.offsetWidth,
-        logPaneWidth = Math.min(Math.max(event.x, 0), maxWidth);
```

Comment:
I removed the `Math.max` because `event.x` was returning a negative value whenever I released the mouse, it would produce a negative value therefore `logPane.style.width` would always end up at `0`. Also I had to add a `min-height` and `min-width` css values for `logPane` and `consolePane` otherwise the flex box was doing weird things when they needed to scroll